Data/models/Bookmark+Sync.swift

```diff
@@ -108,4 +108,16 @@ extension Bookmark {
         DataController.save(context: context)
         Sync.shared.baseSyncOrder = nil
     }
+    
+    class func setOrderForAllBookmarksOnGivenLevel(parent: Bookmark?, forFavorites: Bool, context: NSManagedObjectContext) {
```

Comment:
Although this function name is pretty clear, would still be good to add some function comments.